### DISCUSSION TITLE
Welcome to seistorch Discussions!

### DISCUSSION THREAD

**Discussion by GeophyAI** [GeophyAI/seistorch#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ShiboMao** [comment]
hello ,shaowinw,I used your seistorch, it is really useful. Now, I have a problem. When I do FWI, I want to output the loss curve. How can I do it? If you answer my question, you are a good person, I wish you a peaceful life！
